src/music/win32_m.cpp

```diff
@@ -370,6 +370,26 @@ const char *MusicDriver_Win32::Start(const char * const *parm)
 
 	int resolution = GetDriverParamInt(parm, "resolution", 5);
 	int port = GetDriverParamInt(parm, "port", -1);
+	const char *portname = GetDriverParam(parm, "portname");
+
+	/* Enumerate ports either for selecting port by name, or for debug output */
+	if (portname != nullptr || _debug_driver_level > 0) {
+		uint numports = midiOutGetNumDevs();
+		DEBUG(driver, 1, "Win32-MIDI: Found %d output devices:", numports);
+		for (uint tryport = 0; tryport < numports; tryport++) {
+			MIDIOUTCAPS moc{};
```

Comment:
is there a particular need for {} ?

Reply:
It's probably not strictly required, mainly for being 110% sure nobody complains about passing pointers to uninitialised memory.